Extensions/xkit_patches.js

```diff
@@ -414,6 +507,16 @@ XKit.tools.getParameterByName = function(name){
 
 		};
 
+		/**
+		 * Show an XKit alert window
+		 * @param {String} title - Text for alert window's title bar
+		 * @param {String} msg - Text for body of window, can be HTML
+		 * @param {"error"|others} icon - Window's icon type, determined by CSS class `icon`.
```

Comment:
probably should either put all the possible values in here or make a constant for it or something.